index.html

```diff
@@ -11,17 +11,18 @@
     <title>Microverse Porfolio</title>
 </head>
 <body>
-    <div class="nav">
+    <nav class="nav">
         <div class="navigation">
-            <div class="logo">Didier Ganthier</div>
+            <a class="logo" href="#">Didier Ganthier</a>
             <ul class="links">
                 <li><a href="#">Porfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
             </ul>
+            <img class="hamburger" src="images/hamburger.png" alt="">
         </div>  
-    </div>
-    <div class="header">
+    </nav>
+    <section class="header">
         <div class="headline">I'm Didier<br/>Glad to see you!</div>
         <div class="headline-description">I'm a software developer. I can help you  build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, donn't hesitate to contact me.</div>
```

Comment:
- [ ] [OPTIONAL] This `div` tag also should be replaced with `p` tag for semantic reasons.